src/OpenTelemetry.Exporter.Geneva/Metrics/Transport/MetricEtwDataTransport.cs

```diff
@@ -9,14 +9,17 @@ namespace OpenTelemetry.Exporter.Geneva;
 [EventSource(Name = "OpenTelemetryGenevaMetricExporter", Guid = "{edc24920-e004-40f6-a8e1-0e6e48f39d84}")]
 internal sealed class MetricEtwDataTransport : EventSource, IMetricDataTransport
 {
+    private static MetricEtwDataTransport singleton;
+
     private readonly int fixedPayloadEndIndex;
+    private bool isDisposed;
 
     static MetricEtwDataTransport()
     {
-        Shared = new();
+        singleton = new();
     }
 
-    public static readonly MetricEtwDataTransport Shared;
+    public static MetricEtwDataTransport Shared { get => singleton; }
```

Comment:
Need to change the variable names as "singleton" and "shared" are too broad/general.

Reply:
I can change the name to `MetricEtwDataTransportSingleton` or `MetricEtwDataTransportInstance`, but the the usage will be `this.metricDataTransport = MetricEtwDataTransport.MetricEtwDataTransportSingleton;`, which seems too redundant.

I can rename it to `Instance` as it is used many times as the name for a singleton in the opentelemetry-dotnet repo: https://github.com/search?q=repo%3Aopen-telemetry%2Fopentelemetry-dotnet+instance&type=code.

Reply:
"Instance" is fine IMHO.

Reply:
I've updated it.

Reply:
Do we need to have the field? I think just having the attribute is good enough (e.g. follow https://github.com/search?q=repo%3Aopen-telemetry%2Fopentelemetry-dotnet+instance&type=code)

Reply:
I need to set it to null, so I need to add a setter, which is only needed as `private`.

`get; private set;` can do exactly that...